pkg/resource/node_resource_manager.go

```diff
@@ -130,7 +130,6 @@ func (o *NodeResourceManager) Run(stop <-chan struct{}) {
 			case state := <-o.stateChann:
 				o.state = state
 				o.lastStateTime = time.Now()
-			case <-tspUpdateTicker.C:
```

Comment:
Why delete this case ?

Reply:
cause the origin logic is
1. collect the state from node by 10s interval
2.  compute node resource by 20s interval

The map is not synced, so the agent crashes frequently with `concurrent access to the map` error. 
By thinking deeper, I think node resource controller do not need to trigger the reconcile by interval, as long as there is new message from the channel, it should do reconcile. this simplifies  the flow.